transformations/mr_value_replacement/transformation.py

```diff
@@ -0,0 +1,88 @@
+import torchtext.vocab
+import torch
+from torchtext.data import get_tokenizer
+from interfaces.KeyValuePairsOperation import KeyValuePairsOperation
+import spacy
+from itertools import product
+import random
+from random import sample
+from tasks.TaskTypes import TaskType
+
+class MRValueReplacement(KeyValuePairsOperation):
+    tasks = [TaskType.E2E_TASK]
+    languages = ["en"]
```

Comment:
Please add the relevant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) too.

Reply:
Sorry, this is the first time contributing and I am not sure where I should put the relevant keywords? should be in the PR? or in the code?

Reply:
You should add them in a separate variable called "keywords" like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/2eda3b3552597519aa5e88011e81f594afe4d5ee/transformations/add_hashtags/transformation.py#L92)

Reply:
Done!